pkg/config/store.go

```diff
@@ -56,6 +58,8 @@ type PolicyControllerConfig struct {
 	NoMatchPolicy string `json:"no-match-policy"`
 	// FailOnEmptyAuthorities configures the validating webhook to allow creating CIP without a list authorities
 	FailOnEmptyAuthorities bool `json:"fail-on-empty-authorities"`
+	// PolicyResyncPeriod configures the interval for triggering ClusterImagePolicy reconcile
```

Comment:
Because this will require a controller restart and adding it here is not what this is for (as tried to indicate in the comment), I don't think it should be added here. I think it will be confusing for users to distinguish which fields take effect after a restart and which "immediately".

Reply:
I was also concerned about this as well. Sure thing, I'll follow similarly to how we have "tuf-root" and "tuf-mirror" args and create another one for resync period.